examples/speculative_decoding/distributed_generate/worker_multimodal.sh

```
@@ -0,0 +1,265 @@
+    for pid in "${SERVER_PIDS[@]}" "${MEDIA_HTTP_PID:-}"; do
+        [ -n "${pid:-}" ] && wait "$pid" 2>/dev/null || true
+    done
+}
+
+trap cleanup EXIT
+trap 'exit 130' INT
+trap 'exit 143' TERM
+
+if [ "$API_MODE" = "openai" ]; then
+    python3 -m http.server "$MEDIA_HTTP_PORT" --bind 127.0.0.1 --directory / \
+        >/tmp/multimodal_media_http_${MEDIA_HTTP_PORT}.log 2>&1 &
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)